src/data/tribers/sarika.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 'Sarika'
+words:
+  - Front-end web development
+  - Learning
+  - Animals
+image: images/sarika.jpg
+story: stories/sarika.md
```

Comment:
You won't need this part. The stories are the ones on the front page of the site with their text stored in the `/data/tribers/stories` folder. 

For now we have only written for 3 people in the company about journey to get here.

Reply:
We have a ticket for more people to be featured on the front page, but not yet 🙂 
https://github.com/neontribe/www/issues/270

Reply:
thank you for letting me know